### DISCUSSION TITLE
Chosen multiselct issue after submit

### DISCUSSION THREAD

**Discussion by hadesunseenn** [livewire/livewire#4697] [1. Help]
Hi,
Followed this article and was successfully able to add multi-select in the livewire component.
[https://forum.laravel-livewire.com/t/select-and-select-multi/1120](https://forum.laravel-livewire.com/t/select-and-select-multi/1120)
Used Choices multi-select as mentioned in the above article.
The issue is when we submit form the Choice script is not able to find element and it appears as simple HTML multi-select.
I have created a short video of this issue
[https://drive.google.com/file/d/18e87O-Xy_WMDb6wu66aaY-4hiC_uw6at/view](https://drive.google.com/file/d/18e87O-Xy_WMDb6wu66aaY-4hiC_uw6at/view)
If anyone faced such an issue, please let me know what I am missing. 
Thanks

**Comment by joshhanley** [comment]
@hadesunseenn due to the way Livewire works, any third party libraries that modify the DOM will be wiped out.

Because of this you need to make use of `wire:ignore`, see here for more details https://laravel-livewire.com/docs/2.x/alpine-js#ignoring-dom-changes

I'd do a search and see if you can find someone else's implementation of using Choices with Livewire as I'm sure someone will have done it already.

Hope this helps!

**Comment by hadesunseenn** [comment]
@joshhanley even if I use `wire:ignore` in the component, it still gets wiped out

**Comment by joshhanley** [comment]
@hadesunseenn try wrapping the component in a `div` and putting the `wire:ignore` on that `div` instead.

Hope this helps!
